composer.json

```diff
@@ -25,15 +25,16 @@
     },
     "scripts": {
         "format": "php vendor/bin/phpcs -d date.timezone=UTC --standard=phpcs.xml",
-        "test": "php vendor/bin/phpunit"
+        "test": "php vendor/bin/phpunit",
```

Comment:
Should we exclude vendor directory here (or `phpunit.xml`)?

Reply:
is https://github.com/cloudflare/Cloudflare-WordPress/blob/master/phpunit.xml#L5 what you mean? if so, we've already got that handled (same for phpcs too - https://github.com/cloudflare/Cloudflare-WordPress/blob/master/phpcs.xml#L3).